house/house.go

```diff
@@ -0,0 +1,369 @@
+package house
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/beaujr/nmap_prometheus/assistant"
+	"github.com/beaujr/nmap_prometheus/notifications"
+	pb "github.com/beaujr/nmap_prometheus/proto"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/robfig/cron/v3"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+type networkId struct {
+	Ip  string `json:"ip",yaml:"ip"`
+	Mac string `json:"mac",yaml:"mac"`
+}
+
+type device struct {
+	Id       networkId `json:"id",yaml:"id"`
+	LastSeen int64     `json:"lastSeen",yaml:"lastSeen"`
+	Away     bool      `json:"away",yaml:"away"`
+	Name     string    `json:"name",yaml:"name"`
+	Person   bool      `json:"person",yaml:"person"`
+	Command  string    `json:"command",yaml:"command"`
+	Smart    bool      `json:"smart",yaml:"smart"`
+}
+
+//IOT iotDevices
+var timeAwaySeconds = flag.Int64("timeout", 300, "")
+var houseDevices = []*device{}
+var iotDevices = []*device{}
+var syncStatusWithGA = time.Hour.Seconds()
+var metrics map[string]prometheus.Gauge
+var debug = flag.Bool("debug", false, "Debug mode")
+
+var (
+	people_home = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "home_detector_people_home",
+		Help: "The total number of houseDevices at home",
+	})
+)
+
+type HomeManager interface {
```

Comment:
warning: exported type HomeManager should have comment or be unexported (golint)